pkgs/applications/misc/nwg-displays/default.nix

```diff
@@ -8,6 +8,7 @@
 , pango
 , python310Packages
 , wrapGAppsHook
+, hyprlandSupport ? false, wlr-randr
```

Comment:
is this truly limited to hyprland or is it just a general requirement? wlr-randr isn't hyprland specific